### ISSUE TITLE
Include Dark/Light mode and Mobile/Desktop in "📋 Copy failure details" button

### ISSUE THREAD

**Issue by mgifford** [mgifford/open-scans#173]
These can be added:
https://mgifford.github.io/open-scans/reports/issues/issue-169/2026-03-16T17-26-58-051Z/report.html

> Title: WCAG 2.5.8: Interactive elements have a sufficient target size
> Tags: Accessibility, WCAG 2.5.8, SIA-R111
> 
> Issue: Interactive elements have a sufficient target size (SIA-R111)
> 
> Target application: NSF.gov — - https://nsf.gov/
> 
> Element path: /button[@class="usa-accordion__button usa-nav__link "]
> 
> Snippet: <button type="button" class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-mega-nav-section-1">...</button>
> 
> Related paths: 
> 
> How to fix: 
> Target has insufficient size
> 
> Environment: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/146.0.0.0 Safari/537.36
> 
> ====
> 
> This accessibility issue was found using ALFA and https://github.com/mgifford/open-scans/
